### PR TITLE
Wiki trait links.

### LINKED ISSUE #6742: Trait Wiki Links
For each trait that Requires<> we have `Requires foo, bar.`

Make `foo` and `bar` direct links.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/5244478-trait-wiki-links?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by phrohdoh**
Closes #6742.

### CODE REVIEW

File: OpenRA.Mods.Common/UtilityCommands/ExtractTraitDocsCommand.cs [hunk 1]
```diff
@@ -62,7 +62,7 @@ public void Run(ModData modData, string[] args)
 					{
 						var n = require.Name;
 						var name = n.EndsWith("Info") ? n.Remove(n.Length - 4, 4) : n;
-						doc.Append("`{0}`{1}".F(name, i + 1 == reqCount ? ".\n" : ", "));
+						doc.Append("[{0}](#{1}){2}".F(name, name.ToLowerInvariant(), i + 1 == reqCount ? ".\n" : ", "));
```
Comment (obrakmann) [inline OpenRA.Mods.Common/UtilityCommands/ExtractTraitDocsCommand.cs]: You can still keep the formatting with `doc.Append("[`{0}`](#{1}){2}".F(name, name.ToLowerInvariant(), i + 1 == reqCount ? ".\n" : ", "));`

**Comment by obrakmann** [comment]
:+1: Would be nice if you adopted my suggestion above, but it's not required.

**Comment by RoosterDragon** [comment]
:+1:

File: OpenRA.Mods.Common/UtilityCommands/ExtractTraitDocsCommand.cs (hunk 1 above)
Comment (Phrohdoh) [inline OpenRA.Mods.Common/UtilityCommands/ExtractTraitDocsCommand.cs]: Will do, just allow me more lazy-time.

**Comment by Phrohdoh** [comment]
Updated.

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/b530d1af824a14637ddf7043830d15de0aaa5211%5E...b530d1af824a14637ddf7043830d15de0aaa5211)

**Pull request merged**